android/lib/network/annotation/post_annotation_service.dart

```diff
@@ -46,8 +42,53 @@ Future<bool> postAnnotation(Map<String, dynamic> annotation) async {
     );
     if (response.statusCode != 200) {
       return false;
+    } else {
+      return true;
+    }
+  } catch (err) {
+    if (kDebugMode) {
+      print(err);
     }
-    else {
+    return false;
+  }
+}
+
+Future<bool> postTextAnnotation(Map<String, dynamic> annotation) async {
+  var uuid = const Uuid();
+
+  Response response;
+
+  try {
+    Map<String, dynamic> body = {
+      "@context": "http://www.w3.org/ns/anno.jsonld",
+      "type": "Annotation",
+      "creator": "$serverIP/profile/${annotation['creator']}",
+      "body": [
+        {
+          "type": "TextualBody",
+          "value": annotation['text'],
+          "purpose": "commenting"
+        }
+      ],
+      "target": {
+        "source": annotation['source'],
+        "selector": {
+          "type": "TextPositionSelector",
+          "start": annotation['start'],
+          "end": annotation['end'],
+        }
+      },
+      "id": "c_#${uuid.v4()}"
```

Comment:
Could you please update this as follows:

- For art items: `a{id}`
- For events `e{id}`
- Fo discussion posts `d{id}`

Reply:
I have made the requested changes in the commit https://github.com/bounswe/bounswe2022group7/pull/622/commits/a98e85288c19e389f381ea0f93a6df5a7e7d492f.